composer.json

```diff
@@ -28,7 +28,7 @@
         "intervention/image": "^2.5",
         "jenssegers/agent": "^2.6",
         "laravel/framework": "^10.0",
-        "laravel/passport": "^11.2",
+        "laravel/passport": "^12.0",
```

Comment:
With this upgrade a Test is broken, because the new console interaction isn't supported here. I've created a PR at passport to fix this - we need to wait until this is approved and rolled out.

Reply:
Moved the test to the api scope so it should work now.